packages/corvid-e2e-tests/test/sanity.e2e.js

```diff
@@ -67,4 +68,44 @@ describe("browser sanity", () => {
       await openEditorCliCommand.waitForCommandToEnd();
     })
   );
+
+  test("should show a warning message when closing with unsaved changes and allow to continue working", async () => {
+    process.env.SKIP_UNSAVED_DIALOG = true;
```

Comment:
do not change it on the global process variable.
instead, pass this to the relevant driver command so it'll execute the child process with this variable

Reply:
done